packages/google_maps_flutter/google_maps_flutter_web/example/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
 dev_dependencies:
   google_maps: ^3.4.4
   http: ^0.13.0
-  mockito: ^4.1.4 # Update to ^5.0.0 as soon as this migrates to null-safety
+  mockito: 4.1.1+1 # Update to ^5.0.0 as soon as this migrates to null-safety
```

Comment:
Why did you remove the ^? Wouldn't just lowering the minimum be enough, without pinning an *exact* version?

Reply:
ahh, my bad

Reply:
fixed

Reply:
Thank! (You'll need to merge in master to get the tests fixed; flutter/flutter's switch to NNBD-by-default for `create` broke our CI)